tests/src/rules/first.js

```diff
@@ -35,7 +42,37 @@ ruleTester.run('first', rule, {
                   'use directive';\
                   import { y } from 'bar';"
          , errors: 1
+         , output: "import { x } from 'foo';\
+                  import { y } from 'bar';\
+                  'use directive';"
          })
+  , test({ code: "var foo = bar;\
+                  import { x } from './foo';\
+                  import { y } from './bar';\
+                  import { z } from './baz';"
+         , errors: 3
+         , output: "import { x } from './foo';\
+                  import { y } from './bar';\
+                  import { z } from './baz';\nvar foo = bar;"
+  })
+  , test({ code: "var a = x;\
+                  import { x } from './foo';\
+                  import { y } from './bar';\
+                  import { z } from './baz';"
+         , errors: 2
+         , output: "import { y } from './bar';\
+                  import { z } from './baz';\nvar a = x;\
```

Comment:
this is actually incorrect; because `foo` might depend on side effects from `bar` and `baz` - this example can't be reordered at all.